scripts/exec

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+from github import Github
+
+# Get inputs from shell
+(token, organization_name, path) = sys.argv[1:4]
+
+# Fetch organization from bot repo, then get all organization repos
+organization = Github(token).get_organization(organization_name)
+org_repos = organization.get_repos()
+
+for repo in org_repos:
+    print("Organizing " + repo.name)
+
+    # Automatically merge PRs that meet criteria
+    prs = repo.get_pulls()
+    for pr in prs:
+        pr_reviews = pr.get_reviews()
+        
+        # Parse reviews for merge criteria.
+        changes_requested = False
+        approving_reviews = 0
+        for review in pr_reviews:
+            if review.state == "APPROVED":
+                approving_reviews += 1
+            if review.state == "CHANGES_REQUESTED":
+                changes_requested = True
+        
+        # If all merge criteria pass, merge the PR.
+        if pr.mergeable == True and approving_reviews >= 2 and changes_requested == False:
+            print("Merging PR " + str(pr.number) + " from " + repo.name)
```

Comment:
For this, and all other strings, use a [format specifier](https://pyformat.info) or [f-string](https://realpython.com/python-f-strings/) (preferred).